central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -219,6 +222,20 @@ func (m *managerImpl) processRequestToSensor(ctx context.Context, scanRequest *s
 		return nil, errors.Wrapf(err, "Unable to create scan configuration named %q.", scanRequest.GetScanConfigName())
 	}
 
+	// Check if there are any Existing ScanSettingBindings for the clusters that already have the scan configuration
```

Comment:
I don't think we can do this here.  Update uses this path as well and the SSB will certainly exist for an update.